Makefile

```diff
@@ -22,6 +22,11 @@ k3d-up:
 k3d-down:
 	bash bin/k3d.sh down
 
+restart:
```

Comment:
Can we name this `k3d-rebuild` instead? Just want it to be consistent with the other k3d targets.

Reply:
will do!